app/uploaders/exported_user.rb

```diff
@@ -10,10 +10,13 @@ def store_dir
   end
 
   def extension_allowlist
-    %w[gz]
+    %w[gz zip json]
   end
 
   def filename
-    "#{model.username}_diaspora_data_#{secure_token}.json.gz" if original_filename.present?
+    if original_filename.present?
```

Comment:
Style/GuardClause: Use a guard clause (`return unless original_filename.present?`) instead of wrapping the code inside a conditional expression.